### ISSUE TITLE
Documentation error in frontscope run from source instructions

### ISSUE THREAD

**Issue by katestange** [numberscope/frontscope#155]
In [doc/running-from-source.md](https://github.com/numberscope/frontscope/blob/main/doc/running-from-source.md), in the first section, item 4, it should say to run `npm run prepare` not `npm prepare`.

**Comment by katestange** [comment]
Maybe while we're in there, we should add a note in item 3 that the file `.env.local` ought to be in your root frontscope directory, not some other weird place (as I once did and had to have Liam rescue me...).  I managed to make this error because my other git and npm commands can run regardless of where in the tree I am, so I forgot where I was.

**Comment by gwhitney** [comment]
I am a little confused. `npm install` should automatically run the `prepare` script. Is that not happening for you, Kate? But you are right, if you need to do `prepare` manually, it should be `npm run prepare`, so we might as well fix the docs and your other point too.

**Comment by katestange** [comment]
Nope, I didn't need that command, but I noticed it was wrong.  So it's a separate question whether it should be in the docs at all, and I don't know the answer to that.

**Comment by liammulh** [comment]
https://github.com/numberscope/frontscope/pull/148/commits/59fd25623bc1f78bae3e0e097eb8fbecb53e6926 changes `npm prepare` to `npm run prepare`.

**Comment by katestange** [comment]
I think this is no longer relevant because of #161 ?

**Comment by gwhitney** [comment]
Well, I see a couple remaining points
1) The onboarding doc suggests you edit .env. You shouldn't, you should create a .env.local to override it so that your personal env won't affect the repository when you commit changes.
2) It doesn't say where .env is, as you requested.
3) It doesn't say how to check whether husky ran, as you requested. That's a good idea. But I am wondering why we are bothering to bring up such an unlikely event in a basic onboarding document? I don't think anyone has ever had a problem with husky running, and there is no reason to think there will be a problem. So I propose removing all mention of husky.

If there's agreement on (1-3) I am happy to submit a tiny PR for them.

**Comment by katestange** [comment]
I suggested we simply take out any reference to husky when I reviewed the PR and Liam agreed.

**Comment by katestange** [comment]
And I think the onboarding doc doesn't suggest you edit .env anymore.

**Comment by gwhitney** [comment]
No, #161 is already merged, but all three comments I just made were based on looking at current doc/running_from_source.md in main.  But I did err when I spoke of it as the onboarding doc. I was confusing it with onboarding.md. But I still think about doc/running_from_source.md that

1) It should suggest creating a .env.local alongside .env, not modifying .env.local
2) It should also not mention about husky, because there is no reason to think it will fail. (Or since it is the in-depth one, we could leave in mention of husky but mention how to check it succeeded, which is .git/config should set `hooksPath = .husky`. But frankly I think the chance of failure is so low that we shouldn't bother. I mean, we don't talk about what to do if the git clone fails, but that's probably more likely since maybe someone's internet connection isn't great?)

Let me know if you agree and I'll make a tiny PR.
